### DISCUSSION TITLE
Development & Production Databases for Astro DB

### DISCUSSION THREAD

**Discussion by zellwk** [withastro/roadmap#1066] [Proposal]
### Astro Info

```block
Astro                    v4.16.13
Node                     v22.11.0
System                   macOS (arm64)
Package Manager          npm
Output                   hybrid
Adapter                  @astrojs/node
Integrations             astro-expressive-code
                         astro:db
                         @astrojs/db/file-url
                         @astrojs/tailwind
                         @astrojs/svelte
                         @astrojs/mdx
                         @astrojs/sitemap
```

### If this issue only occurs in one browser, which browser is a problem?

NA

### Describe the Bug

Not a bug, more of a feature. 

I propose allowing users to add development & production databases in `.env.development` or `.env.production`. By doing so, we can test against race conditions that may not be present on a local environment. 

Right now, `astro db push --remote` gets the `ASTRO_DB_URL` from `.env` only. 

### What's the expected result?

Allow `astro db push --remote`  to read from `.env.development` and `.env.production`. 

### Link to Minimal Reproducible Example

NA

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by ascorbic** [comment]
I've moved this to roadmap discussions, because it's a feature request

**Comment by florian-lefebvre** [comment]
Astro DB has been deprecated
